physics/cu_gf_deep.F90

```diff
@@ -4146,6 +4162,7 @@ subroutine cup_output_ens_3d(xff_mid,xf_ens,ierr,dellat,dellaq,dellaqc,  &
 !       used in cup_forcing_ens (including screening of some
 !       closures over water) to properly normalize xmb
            clos_wei=16./max(1.,closure_n(i))
+         clos_wei=1.
```

Comment:
Is it intentional to leave in the calculation of `clos_wei` above this line since you're resetting it to 1 here?

Reply:
@grantfirl Thanks for pointing it out. This should be resolution dependent, and will address it soon.